### ISSUE TITLE
Test

### ISSUE THREAD

**Issue by kongenfloat** [technocake/SigmaAlfa2#2]
Create test directory with the test functionality on the sigma api

**Comment by Arxcis** [comment]
Det hadde vert kult dersom test-systemet ble laget slik at det gikk ann å kjøre ALLE tester, vha. bare 1 kommando - 1 linje i koden.

**Comment by Murplugg** [comment]
@Arxcis Sounds like Automated Unit testing.

**Comment by technocake** [comment]
@Dynode Ja, det finns sikkert noe python-test-rammeverk som gjør det mulig å få integrert med Visual Studios. Kan du sjekke opp det?

@Arxcis Det er slik idag: python test_sigma.py --> alle tester kjører.
